src/platform/forms-system/src/js/review/ObjectField.jsx

```diff
@@ -207,7 +207,16 @@ class ObjectField extends React.Component {
       );
     }
 
-    return <>{renderedProperties}</>;
+    return isReactComponent(ObjectViewField) ? (
+      <ObjectViewField
+        {...this.props}
+        renderedProperties={renderedProperties}
+        title={title}
+        defaultEditButton={defaultEditButton}
+      />
+    ) : (
+      <>{renderedProperties}</>
+    );
```

Comment:
The code from the `isRoot` check down to here is _similar_ to [another pattern from not too long ago](https://github.com/department-of-veterans-affairs/vets-website/pull/18096#discussion_r684534945). That one had four paths with two possible outcomes, but here we have four paths with three possible outcomes.

What we might be able to do is flip the condition like this:

```jsx
if (isReactComponent(ObjectViewField)) {
  return (
     <ObjectViewField
        {...this.props}
        renderedProperties={renderedProperties}
        title={title}
        defaultEditButton={defaultEditButton}
      />
   );
}

return isRoot ?
  (
      <>
        {!formContext?.hideHeaderRow && (
          <div className="form-review-panel-page-header-row">
            {title?.trim() &&
              !formContext?.hideTitle && (
                <h4 className="form-review-panel-page-header vads-u-font-size--h5">
                  {title}
                </h4>
              )}
            {defaultEditButton()}
          </div>
        )}
        <Tag className="review">{renderedProperties}</Tag>
      </>
    ) : (
    <>{renderedProperties}</>
    );
```

Reply:
Yeah, this should work great. If it passes the tests (which I expect it will), I'm all in favor of this simplification.